examples/point-of-sale/package.json

```diff
@@ -6,7 +6,7 @@
     "repository": "https://github.com/solana-labs/solana-pay",
     "license": "Apache-2.0",
     "engines": {
-        "node": ">=16"
+        "node": "18"
```

Comment:
Please don't change the Node version.

Reply:
Either you select "16", or you select "18", but setting ">=16" trigger a warning when building. Using vercel, the app build with whatever node version is in the setting. If you don't like "18", set "16" insead, please.